### PR TITLE
Update what-are-analysis-environments.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ category: must-know
...
+To open the Python environment in your terminal (known as *initializing the [interpreter](https://www.enki.com/glossary/general/interpreter)*), type in the word `python` and press Enter.
 
-> 💡 Different versions of Python will have different keywords to launch the basic interpreter. However, the most common ones are `py` and `python`.
+> 💡 Different versions of Python will have different keywords to launch the interpreter. The most common ones are `py` and `python`.
 
-Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended way for coding. This is because there are far superior environments with better interpreters for coding in Python.
+Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended tool for codinng as it lacks many features.
 
-Some of these are:
+There are far superior environments with better interpreters for coding in Python, such as
```
Comment (nem035) [inline python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md:27]:
```suggestion
There are far superior environments with better interpreters and more powerful features, such as:
```

File: python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md [hunk 2]
```diff
@@ -14,17 +14,17 @@ category: must-know
...
+For instance, the most basic environment you can write Python in is your OS's terminal[1]. 
 
-You can write and test code here by first initializing the [interpreter](https://www.enki.com/glossary/general/interpreter) by typing `python` in your terminal.
+To open the Python environment in your terminal (known as *initializing the [interpreter](https://www.enki.com/glossary/general/interpreter)*), type in the word `python` and press Enter.
 
-> 💡 Different versions of Python will have different keywords to launch the basic interpreter. However, the most common ones are `py` and `python`.
+> 💡 Different versions of Python will have different keywords to launch the interpreter. The most common ones are `py` and `python`.
 
-Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended way for coding. This is because there are far superior environments with better interpreters for coding in Python.
+Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended tool for codinng as it lacks many features.
```
Comment (nem035) [inline python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md:25]:
```suggestion
Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended tool for coding as it lacks many features.
```

**Comment by nem035** [comment]
Can we also make "shells" a glossary entry or a footnote?

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by Stefan-Stojanovic — APPROVED** [review]

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md?ref=4d67a657df3f47f69ad427f86417645c7dfa3a28)

**Review by Stefan-Stojanovic — APPROVED** [review]

**Pull request merged**